_pages/user-experience/agency-logo.md

```diff
@@ -32,17 +32,21 @@ Your agency logo will appear to the right of the Login.gov logo in your integrat
 <div class="grid-row">
   <div class="float-left agency-logo-width">
     <img src="{{ site.baseurl }}/assets/img/do_logo.svg" alt="An example agency logo in black on a white background" class="display-block green-bottom-border">
-      <div class="text-green float-left">
-        <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"/></svg>
-      </div>
-      <div class="margin-left-3">
-        <span class="text-uppercase text-bold text-green">Do</span>: Add a logo that is high contrast on a transparent background.
-      </div>
+    <div class="text-green float-left">
+      <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <use xlink:href="{{site.baseurl}}/assets/img/sprite.svg#check_circle"></use>
+      </svg>
```

Comment:
should we make this `<svg ... >`  bit a partial and use it inside of the other partial? That one could be like "list-icon.html" or something?

Reply:
updated in https://github.com/18F/identity-dev-docs/pull/449/commits/c6da45bc3dfc11deca6f45fe585ff7bb66df0c1a